scripts/dvm

```diff
@@ -177,6 +177,7 @@ dvm_use() {
     echo 'export DVM_ROOT; DVM_ROOT="$DVM_ROOT"' > "$defaults"
     echo "export DART_SDK; DART_SDK=\"$DVM_ROOT/darts/$version\"" >> "$defaults"
     echo "PATH=\"\$DVM_ROOT/darts/$version/bin:\$PATH\"" >> "$defaults"
+    ln -sfn -- "$DVM_ROOT/darts/$version" "$DVM_ROOT/darts/default"
```

Comment:
Given that we source `$DVM_ROOT/environments/default` at the end of the script, why is a symlink necessary? If the user wants to create a named alias, the `alias` command is there to do so.

Generally, the default is meant to point to some existing version or alias. Aliases are meant to allow for project/environent-specific friendly names that can be bumped over time. e.g. `foo-project` and `bar-project` each rely on some different version of Dart. 

On the surface, creating a `default` alias seems a bit tautological, but I'm curious if this is in response to some problematic behaviour.

Reply:
I've just realized that what I did was essentially replicating the `alias` command :facepalm: I've removed commit that added this line